src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -4502,6 +4502,17 @@ namespace HeatBalanceManager {
                                     ", missing material = " + ConstructAlphas(Layer));
                     ErrorsFound = true;
                 } else {
+                    MaterialLayerGroup = Material(Construct(TotRegConstructs + ConstrNum).LayerPoint(Layer)).Group;
+                    if (!((MaterialLayerGroup == GlassEquivalentLayer) || (MaterialLayerGroup == ShadeEquivalentLayer) ||
+                          (MaterialLayerGroup == DrapeEquivalentLayer) || (MaterialLayerGroup == BlindEquivalentLayer) ||
+                          (MaterialLayerGroup == ScreenEquivalentLayer) || (MaterialLayerGroup == GapEquivalentLayer))) {
+                        ShowSevereError("Invalid material layer type in window " + CurrentModuleObject + " = " +
+                                        Construct(TotRegConstructs + ConstrNum).Name);
+                        ShowContinueError("...Window layer material type = " + ConstructAlphas(Layer) +
+                                          " is not allowed in Construction:WindowEquivalentLayer window object.");
```

Comment:
@Nigusse This is good, except `ConstructAlpahs(Layer)` is the material name, not the material type, so the error message is a little confusing:
```
   ** Severe  ** Invalid material layer type in window Construction:WindowEquivalentLayer = SERRAMENTI PVC FINESTRE DOPPIA CAMERA 7CM
   **   ~~~   ** ...Window layer material type = VETRO DOPPIA CAMERA 4-9-4 NON TRATTATO SUPERFIC. is not allowed in Construction:WindowEquivalentLayer window object.
```
Maybe just delete "material type" above. And add "Only materials of type Material:*:EquivalentLayer are allowed."

Reply:
But then the user has to identify which material is wrong. Small effort I know but wouldn't it be better to point to it?

Reply:
My suggestion was to keep the name, just delete two words:
** ... Window layer ~material type~ = VETRO VETRO DOPPIA CAMERA 4-9-4 NON TRATTATO SUPERFIC. is not allowed ....
**  Only materials of type Material:*:EquivalentLayer are allowed.